dapps/marketplace/src/pages/identity/OAuthAttestation.js

```diff
@@ -213,7 +247,29 @@ class GoogleAttestation extends Component {
   }
 }
 
-export default withRouter(GoogleAttestation)
+export default withIsMobile(withRouter(OAuthAttestation))
 
-require('react-styl')(`
-`)
+// require('react-styl')(`
```

Comment:
We don't need this anymore? Can just delete it if not

Reply:
Removed it.